test/src/locator.spec.ts

```diff
@@ -222,7 +222,9 @@ describe('Locator', function () {
     });
 
     it('should time out', async () => {
-      const clock = sinon.useFakeTimers();
+      const clock = sinon.useFakeTimers({
+        shouldClearNativeTimers: true,
```

Comment:
the lack of this config was causing hanging after all tests: apparently, the default behaviour for sinon is not invoke the native clearTimeout if the timer was started before the fake timers were activated.